engaging-networks-rest.gemspec

```diff
@@ -1,79 +1,73 @@
 # Generated by juwelier
 # DO NOT EDIT THIS FILE DIRECTLY
 # Instead, edit Juwelier::Tasks in Rakefile, and run 'rake gemspec'
-# frozen_string_literal: true
```

Comment:
Should we keep this to avoid all those `freeze` calls?

Reply:
This file is auto-generated by `bundle exec rake gemspec`; AFAIK we have no control over the output of that task.